configuration.json

```diff
@@ -2,14 +2,111 @@
   "videoRecord": false,
   "videoDir": "build/cify/videos/",
   "capabilities": {
-    "android": {
-      "capability": "android"
-    },
-    "browser": {
-      "capability": "chrome"
-    },
-    "ios": {
-      "capability": "iphone"
-    }
+    "browser": [
+      {
+        "capability": "chrome",
+        "capabilityId": "ChromeMac"
```

Comment:
Why here is capabilityId ChromeMac and some many chrome capabilities?

Reply:
I think, that this was made to demonstrate, that capabilityId is optional or to test capabilityId generation. Anyways, I've removed those.

Reply:
Ok, all these capabilities are there in testing purposes

Reply:
You can create a separate configuration.json/capabilities.json for testing purposes. It's better to split testing env from prod. You can pass a path to configuration.json via parameter to cucumber:
gradlew cucumber -.....
                                -Ptags=~@TODO -PcapabilitiesFilePath=$SOME_PATH/capabilities.json

Reply:
Implemented that via gradle and env variables